packages/appstore/src/constants/tour-steps-config.tsx

```diff
@@ -166,6 +166,118 @@ export const getWalletStepConfig = (): Step[] => [
         spotlightPadding: 8,
         styles: { spotlight: { borderRadius: '4.8rem' } },
     },
+    {
+        title: (
+            <React.Fragment>
+                <Text as='p' weight='bold' color='brand-red-coral'>
+                    {localize('CFDs trading accounts')}
+                </Text>
+                <div className='toggle-account-type__divider' />
+            </React.Fragment>
+        ),
+        content: (
+            // TODO: update this component if there's a cfd account
+            <Text as='p' size='xs'>
+                {localize(
+                    'This is your CFDs trading account. Click Get to create the trading account you desire for trading.'
+                )}
+            </Text>
+        ),
+        target: '#trading-app-card__derived',
+        disableBeacon: true,
+        disableOverlayClose: true,
+        spotlightPadding: 3,
+        placement: 'right',
+        styles: { spotlight: { borderRadius: '0.6rem', height: '100px' } },
+    },
+    {
+        title: (
+            <React.Fragment>
+                <Text as='p' weight='bold' color='brand-red-coral'>
+                    {localize('Deriv Apps trading account')}
+                </Text>
+                <div className='toggle-account-type__divider' />
+            </React.Fragment>
+        ),
+        content: (
+            <Text as='p' size='xs'>
+                {localize(
+                    'This is your Deriv Apps trading account balance. Click Transfer to move funds between your Wallet and Deriv Apps trading account.'
+                )}
+            </Text>
+        ),
+        target: '.currency-switcher-container',
+        disableBeacon: true,
+        disableOverlayClose: true,
+        placement: 'left',
+        spotlightPadding: 0,
+        styles: {
+            spotlight: { borderRadius: '0.6rem' },
+        },
+    },
+    {
+        title: (
+            <React.Fragment>
+                <Text as='p' weight='bold' color='brand-red-coral'>
+                    {localize('Deriv apps')}
+                </Text>
+                <div className='toggle-account-type__divider' />
+            </React.Fragment>
+        ),
+        content: (
+            <Text as='p' size='xs'>
+                {localize('Choose a Deriv app to trade options or multipliers.')}
+            </Text>
+        ),
+        target: '#trading-app-card__dtrader',
+        disableBeacon: true,
+        disableOverlayClose: true,
+        spotlightPadding: 3,
+        placement: 'right',
+        styles: { spotlight: { borderRadius: '0.6rem', height: '100px' } },
```

Comment:
@aizad-deriv I'm not sure, but do we need to use `height` in `pixels` instead of `rem`?

Reply:
It can be both but you are right I should change it to rem `instead`. Thanks for spotting this 😃 👍🏼